tools/block-generator/run_runner.py

```diff
@@ -0,0 +1,153 @@
+import argparse
```

Comment:
pythonized `run_runner.sh` with some additions:

* more variables exposed as command line args
* ability to `--keep-alive` so can separate the docker standing up from the script runner
* dump script runner args to terminal so can run directly from terminal
* dump vs-code launcher compatible args so can debug the runner in vs-code
* accept `--purge` so can just clean up the docker container

Reply:
TODO: if this change is accepted, add an example usage to the README